api/com/dynatrace/oneagent/sdk/api/infos/TraceContextInfo.java

```diff
@@ -0,0 +1,38 @@
+package com.dynatrace.oneagent.sdk.api.infos;
+
+/**
+ * Provides information about the current PurePath node using the TraceContext
+ * (Trace-Id, Span-Id) model as defined in https://www.w3.org/TR/trace-context.
+ * The Span-Id represents the currently active PurePath node.
+ */
+public interface TraceContextInfo {
+	public static final String INVALID_TRACE_ID = "00000000000000000000000000000000";
+	public static final String INVALID_SPAN_ID = "0000000000000000";
+
+	/**
+	 * @return True if the value returned by {@link getTraceId()} is not equal to
+	 * {@link INVALID_TRACE_ID} and the value returned by {@link getSpanId}
+	 * is not equal to {@link INVALID_SPAN_ID}.
+	 */
+	boolean isValid();
```

Comment:
How would this interact with sampling? Should we expose a `boolean isSampled()`?

Reply:
The scenario for which this API is intended is log-enrichtment. Sampling information is out-of-scope there.